### PR TITLE
Fix typos in `scripts/` and `.github/` subdirectories

### PR DESCRIPTION
**Pull request by luzpaz**
---
name: 🗎 Documentation and Styling
about: Adding or modifying **documentation** or **code style**.

---

**Description**

Fix various typos

**Changelog**

- Fixed typos in scripts/ subdirectory
- Fixed typos in .github/ subdirectory

### CODE REVIEW

File: scripts/wheels/windows/build.ps1 [hunk 1]
```diff
@@ -87,7 +87,7 @@ Write-Host "Starting core build"
 Write-Host "Finished core build"
 
 foreach ($python in $pythons){
-    Write-Host "Begining build for python $($python)"
+    Write-Host "Beginning build for python $($python)"
```
Comment (sunethwarna) [inline scripts/wheels/windows/build.ps1:90]: Line 83 can be changed to:
Core can be `built` independently of the python version.

File: scripts/standard_configure_mac.sh [hunk 2]
```diff
@@ -49,5 +49,5 @@ rm -rf "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}/CMakeFiles"
  -DUSE_EIGEN_MKL=OFF                                                                                        \
  -DKRATOS_GENERATE_PYTHON_STUBS=ON
 
-# Buid
+# Build
```
Comment (sunethwarna) [inline scripts/standard_configure_mac.sh:52]: Line 10 can be changed to following:
You can find a list `with` all the `compilation` options in INSTALL.md or here:

File: scripts/standard_configure_MINGW.sh [hunk 3]
```diff
@@ -48,5 +48,5 @@ cmake ..
 -DKRATOS_GENERATE_PYTHON_STUBS=ON                                                                   \
 -DUSE_EIGEN_MKL=OFF
 
-# Buid
+# Build
```
Comment (sunethwarna) [inline scripts/standard_configure_MINGW.sh:51]: Line 10 can be changed to:
You can find a list `with` all the `compilation` options in INSTALL.md or here:

File: scripts/standard_configure.sh [hunk 4]
```diff
@@ -47,5 +47,5 @@ cmake -H"${KRATOS_SOURCE}" -B"${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}" \
 -DUSE_EIGEN_MKL=OFF                                                 \
 -DKRATOS_GENERATE_PYTHON_STUBS=ON
 
-# Buid
+# Build
```
Comment (sunethwarna) [inline scripts/standard_configure.sh:50]: Line 10 can be changed to:
You can find a list `with` all the `compilation` options in INSTALL.md or here:

File: .github/workflows/intel_configure.sh [hunk 5]
```diff
@@ -68,5 +68,5 @@ ${KRATOS_CMAKE_OPTIONS_FLAGS} \
 -DTRILINOS_LIBRARY_PREFIX="trilinos_" \
 -DCMAKE_UNITY_BUILD=ON
 
-# Buid
+# Build
```
Comment (sunethwarna) [inline .github/workflows/intel_configure.sh:71]: Line 8 can be changed to:
You can find a list `with` all the `compilation` options in INSTALL.md or here:

File: .github/workflows/configure.sh [hunk 6]
```diff
@@ -73,5 +73,5 @@ ${KRATOS_CMAKE_OPTIONS_FLAGS} \
 -DCMAKE_UNITY_BUILD=ON \
 -DINCLUDE_MMG=ON                                    \
 
-# Buid
+# Build
```
Comment (sunethwarna) [inline .github/workflows/configure.sh:76]: Line 8 can be changed to:
You can find a list `with` all the `compilation` options in INSTALL.md or here:

File: .github/workflows/centos_configure.sh [hunk 7]
```diff
@@ -45,5 +45,5 @@ ${KRATOS_CMAKE_OPTIONS_FLAGS} \
 -DCMAKE_CXX_FLAGS="${KRATOS_CMAKE_CXX_FLAGS} -O0 -Wall" \
 -DCMAKE_UNITY_BUILD=ON                              \
 
-# Buid
+# Build
```
Comment (sunethwarna) [inline .github/workflows/centos_configure.sh:48]: Line 8 can be changed to:
You can find a list `with` all the `compilation` options in INSTALL.md or here:

**Review by sunethwarna — DISMISSED** [review]
Minor missed corrections. Otherwise this is ok with me :)... Thanks :)

**Comment by luzpaz** [comment]
@sunethwarna Added requested revisions in [71b5ac3](https://github.com/KratosMultiphysics/Kratos/pull/10517/commits/71b5ac322d951d3d8dbde2e90b6be7c9150a3490)

**Review by sunethwarna — APPROVED** [review]
Nice 👍. Thanks

**Pull request merged**